docs/manual/java/gettingstarted/index.toc

```diff
@@ -2,10 +2,8 @@ IntroGetStarted:Introduction
 JavaPrereqs:Lagom Java prerequisites;next=GettingStartedMaven,GettingStartedSbt
 GettingStartedMaven:Creating and running Hello World with Maven;next=UnderstandHello
 GettingStartedSbt:Creating and running Hello World with sbt;next=UnderstandHello
-UnderstandHello:Understanding Hello World;next=MavenIDEs,EclipseSbt,IntellijSbt
-MavenIDEs:Integrating Maven projects with an IDE;next=LagomExamples
-EclipseSbt:Integrating sbt with Eclipse;next=LagomExamples
-IntellijSbt:Integrating sbt with IntelliJ;next=LagomExamples
+UnderstandHello:Understanding Hello World
 LagomExamples:Lagom examples
+WhatsNext:What's Next?
```

Comment:
I've heard from a number of Australians (including my wife) that the US style of capitalizing most words in a heading feels either overly formal or like shouting. I've had to train myself out of it 😄 

Standards are pretty loose... Google "sentence case vs. title case" if you have some time to kill. There isn't exactly a definitive style guide for US English, there are several that are specific to different fields or industries. In general, title case is always used for things like book titles, sentence case it always used for normal sentences (naturally) and for things like newspaper headlines, chapter titles and headlines it's a lot more varied. I think we could choose either one without looking too strange to Americans, but as you've observed, title case might look more strange to non-Americans.

One advantage of sentence case is that it's less ambiguous. With title case, the rules for what gets capitalized and what is lowercased are more complicated and less universal.